acceptance/cli/cli.go

```diff
@@ -385,8 +385,8 @@ func setupTUF(ctx context.Context, vars map[string]string, environment []string)
 	}
 	vars["TUF"] = tufURL
 
-	vars["CERT_IDENTITY"] = "https://kubernetes.io/namespaces/default/serviceaccounts/default"
-	vars["CERT_ISSUER"] = "https://kubernetes.default.svc.cluster.local"
+	vars["CERT_IDENTITY"] = "luizcarvalho85@gmail.com"
```

Comment:
Indeed. If it is executed as a GitHub Workflow, it would tie to that workflow's identity which I believe is much nicer.

FWIW, this doesn't prevent someone else from regenerating the data that expires (TUF root). In the case where a new image is needed, for whatever reason, and someone else runs the script to do so, then it's simply a matter of updating the identity values as well.

I want the process of renewing the TUF root data dead simple since it needs to happen every 6 months. Right now, I seem to be the bottleneck for performing that process. So in a way, these changes are less tied to me :wink:

Reply:
@lcarva & @zregvart: Would tying this to the enterprisecontractcommunity[at]gmail[dot]com identity be better?

Reply:
Created https://github.com/enterprise-contract/ec-cli/issues/1622 to do this via a workflow.